Makefile

```diff
@@ -5,7 +5,8 @@
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 0.0.1
 
-CONTAINER_ENGINE ?= docker  #Using docker or podman to build and push images
+# Using docker or podman to build and push images
```

Comment:
Why this could fix the issue? Is that because the comment in the same line causing the problem?